gratipay/billing/payday.py

```diff
@@ -354,6 +361,22 @@ def update_balances(cursor):
                 SELECT *, (SELECT id FROM paydays WHERE extract(year from ts_end) = 1970)
                   FROM payday_payments;
         """)
+        # Copy due value back to payment_instructions
+        cursor.run("""
+            UPDATE payment_instructions pi
+               SET due = pi2.due
+              FROM payment_instructions_due pi2
+             WHERE pi.id = pi2.id
+        """)
+        # Reset older due values to zero
```

Comment:
I think this is extraneous. If we use the `current_payment_instructions` view in order to get the `due` amount, then we'll always get the current `due` amount. We'll never use values from old `payment_instructions` records, so it doesn't matter whether they're zero or some other value. Right?